meta/parse.pl

```diff
@@ -2130,8 +2130,6 @@ sub ProcessStructValueType
     return "SAI_ATTR_VALUE_TYPE_UINT32"         if $type eq "sai_label_id_t";
     return "SAI_ATTR_VALUE_TYPE_INT32"          if $type =~ /^sai_\w+_type_t$/; # enum
     return "SAI_ATTR_VALUE_TYPE_NAT_ENTRY_DATA" if $type eq "sai_nat_entry_data_t";
-    return "SAI_ATTR_VALUE_TYPE_MACSEC_SCI"     if $type eq "sai_macsec_sci_t";
-    return "SAI_ATTR_VALUE_TYPE_MACSEC_SSCI"    if $type eq "sai_macsec_ssci_t";
```

Comment:
did you removed all those from sai_attbute_value_t uniion ? and replaced with u64 ?

Reply:
Replaced with u64 for SCI and u32 for SSCI.